toffy/fov_watcher_test.py

```diff
@@ -113,6 +114,12 @@ def test_watcher(run_cbs, fov_cbs, kwargs, validators, add_blank):
             fovs = fovs[1:]
 
         for i, validator in enumerate(validators):
-            validator(os.path.join(tmpdir, f'cb_{i}', RUN_DIR_NAME), fovs)
+            if i <= 1:
```

Comment:
Given that each condition is different, no longer makes sense for this to be a for loop. Should just call each one with the correct arguments